### ISSUE TITLE
We would like to become i18n maintainers for Turkish language

### ISSUE THREAD

**Issue by ismailarilik** [ruby/www.ruby-lang.org#2073]
Hi.

I (@ismailarilik) and Ender Ahmet Yurt (@enderahmetyurt) want to become i18n maintainers for Turkish language. The site is available in Turkish but it is not up-to-date. I sent some PRs(#1985, #1980 and #1979) but they were not merged. I understand that no one existed to approve those changes. So following [Bacome a maintainer](https://github.com/ruby/www.ruby-lang.org/wiki/Become-a-maintainer) guide, I am applying with a team of 2 volunteer person and a rational. You can see my proficiency in both English and Turkish by looking at my translations in PRs I mentioned. @enderahmetyurt can also confirm his voluntariness and show his proficiency.

Thanks...

**Comment by ismailarilik** [comment]
@hsbt

**Comment by hsbt** [comment]
Created www-ruby-lang-i18n-tr team and invite you.

**Comment by ismailarilik** [comment]
@hsbt thanks for your invitation. I noticed that the team name is "www-ruby-lang-i18n-tr" but the pattern is "www-ruby-lang-org-i18n-<lang>" as I saw so shouldn't it be "www-ruby-lang-org-i18n-tr".

The name may be important for these situations:

1. For convenience (for example, trying to find the team with language code)
2. For automatic review requests on PRs (I don't know how PR names match with team names; this may not be a problem.)
